.idea/misc.xml

```diff
@@ -3,4 +3,14 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
+  <component name="UnattendedHostPersistenceState">
+    <option name="openedFilesInfos">
+      <list>
+        <OpenedFileInfo>
+          <option name="caretOffset" value="129" />
+          <option name="fileUrl" value="file://$PROJECT_DIR$/base_app/src/index.html" />
+        </OpenedFileInfo>
+      </list>
+    </option>
+  </component>
```

Comment:
Is there a specific reason why the .idea folder has been committed? Might be worth just deleting the .idea folder entirely and adding it to the gitignore

Reply:
I'd like to keep the  dictionaries folder (or an alternative dictionary) so that `simpleui` and similar aren't highlighted as a misspellings by the IDE.

Updated `.gitignore` with the other filenames that don't need to be checked in.